packages/honeycomb-opentelemetry-web/src/types.ts

```diff
@@ -134,6 +134,11 @@ export interface HoneycombOptions extends Partial<WebSDKConfiguration> {
   /** Config options for web vitals instrumentation. Enabled by default. */
   webVitalsInstrumentationConfig?: WebVitalsInstrumentationConfig;
   globalErrorsInstrumentationConfig?: GlobalErrorsInstrumentationConfig;
+
+  /**
+   * Controls the verbosity of the logs. Defaults to 'DEBUG'. Current options are 'DEBUG', 'INFO', 'WARN', and 'ERROR'.
+   */
+  logLevel?: 'DEBUG' | 'INFO' | 'WARN' | 'ERROR';
```

Comment:
I think we should use the OpenTelemetry provided log levels type ([`DiagLogLevel`](https://github.com/open-telemetry/opentelemetry-js/blob/4947c2d6a4d33343194879c3b3eb18040eb9683b/api/src/diag/types.ts#L72)) instead of creating our own enum.

Reply:
Will switch to this enum, thanks for the callout!